kmir/src/kmir/kdist/mir-semantics/symbolic/p-token.md

```diff
@@ -193,10 +193,18 @@ The code uses some helper sorts for better readability.
   syntax List ::= #asU8s ( Int )            [function]
                 | #asU8List ( List , Int ) [function]
   // -------------------------------------
-  rule #asU8s(X) => #asU8List(.List, X)
+  rule #asU8s(X) => #asU8List(.List, X) [concrete]
```

Comment:
I think this is a good idea. Requires testing, of course, but we want to avoid building up the list of bit-shifts and enable round-tripping, which this should do.